### PR TITLE
Feature/161551726 create an experiment card type

### PR DESCRIPTION
**Pull request by lingyun1010**

### CODE REVIEW

**Comment by monicajianu** [comment]
@lingyun1010 Can you merge the latest changes from master as quite a few things have changed. This is the trouble when you branch from a feature branch, it's not always stable. 
Also, can you add some way example data to the demo `index.html` so that your new card type is displayed?
And my last point... I notice your new `ImageCard` is almost identical to the `SpeciesCard`, except for the image bit. I'm wondering if the code could be refactored to increase reuse/decrease code repetition.

**Comment by lingyun1010** [comment]
Thanks for your review. I have rebased my branch with the master one and added `ImageCard` examples in `index.html`. Please have another review.

As for the last point, I have discussed with @alfonsomunozpomer  before. We would like to keep clean with your codes, so the new card type was produced. If you think we could merge this two card types, we could have a discussion about it together.

**Comment by monicajianu** [comment]
@lingyun1010 You should be careful with rebasing, it seems like it created some duplicate commits! It's a dangerous tool...

**Review by monicajianu — APPROVED** [review]

**Review by monicajianu — CHANGES_REQUESTED** [review]
@lingyun1010 Actually, I just realised you haven't pulled the latest changes from master... The way in which the cards are initialised is different now, it's using a `switch` statement rather than the mapping from types to components. In order to make this branch more understandable, maybe you could branch off master again and cherry pick your commits? It would also solve the duplicate commits and it would leave us with a nice clean history.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
@lingyun1010 Please create a new branch and place the changes related to the story. It’s very hard to follow, however, keep in mind and be extra careful about the following:
• Don’t place `.DS_Store` files in a commit. [I added a few lines in our template package `.gitignore` file](https://github.com/ebi-gene-expression-group/atlas-package/commit/70f68b09680aa555bec4ef5f89da429efd113bbf), but please do the same here.
• Be careful about spacing in function declarations. Use `eslint`, too.
• Don’t use `var`, either `let` or `const`. Only use `let` if necessary.
• Don’t use id in your HTML elements. I think this is because you followed the template in the initial part of the code.

Once you create the new branch close this PR and we can discuss on the other one. Remember to add a short summary of what you did in the PR description.
